src/ORM/Queries/SQLSelect.php

```diff
@@ -564,6 +576,37 @@ public function getUnions(): array
         return $this->union;
     }
 
+    /**
+     * Adds a Common Table Expression (CTE), aka WITH clause.
+     *
+     * Use of this method should usually be within a conditional check against DB::get_conn()->supportsCteQueries().
+     *
+     * @param string $name The name of the WITH clause, which can be referenced in any queries UNIONed to the $query
+     * and in this query directly, as though it were a table name.
+     * @param string[] $cteFields Aliases for any columns selected in $query which can be referenced in any queries
```

Comment:
Have only listed two of the four params and no return, seems like they should all be in the docblock if you're going to use docblock params

Reply:
Haven't we agreed that if there's no additional context for a given parameter, then that parameter should not be added to the docblock? I don't see what benefit adding those parameters would have.

- When hovering over the method in an IDE, all relevant information is displayed.
- When viewing in API docs, all relevant information is displayed.

Reply:
OK can ignore last comment